doc/examples/viz_ui.py

```diff
@@ -1,6 +1,16 @@
-import numpy as np
+"""
+==============
+User Interface 
+==============
 
-from dipy.data import read_viz_icons
+This example shows how to use the UI API.
+Currently includes button, textbox, panel, and line slider.
+
+First, a bunch of imports.
```

Comment:
Remove this comment and check again which of the imports you really need for this tutorial.
Do you really need all these imports? Checking if vtk is available?